pkg/cmd/server/start/node_args.go

```diff
@@ -138,12 +138,7 @@ func (args NodeArgs) BuildSerializeableNodeConfig() (*configapi.NodeConfig, erro
 		config.ServingInfo.ClientCA = admin.DefaultKubeletClientCAFile(args.MasterCertDir)
 	}
 
-	// Roundtrip the config to v1 and back to ensure proper defaults are set.
```

Comment:
does this need to set `BindNetwork` as well? do we support it on the kubelet? do we need to?